tests/data/PortalState.ts

```diff
@@ -114,6 +119,11 @@ export class PortalState {
     return this;
   }
 
+  public createUserSocialMedia(...userSocialMediaList: UserSocialMediaModel[]): PortalState {
+    this.userSocialMediaList = this.userSocialMediaList.concat(userSocialMediaList);
+    return this;
+  }
+
```

Comment:
The pattern we follow in `PortalState` with inserting data that is related to users is to specify the user in the first parameter, and then the list of social media after (e.g. [submitFeedback](https://github.com/acmucsd/membership-portal/blob/master/tests/data/PortalState.ts#L192), orderMerch, attendEvents, etc.). That way, you don't need to pass in the user model into the social media factory, and can instead just pass in the user model into this function and handle the relationship here.